skrub/datasets/_utils.py

```diff
@@ -2,7 +2,38 @@
 from pathlib import Path
 
 
-def get_data_dir(name: str | None = None) -> Path:
+def get_data_home(data_home=None) -> str:
+    """Returns the path of the skrub data directory.
+
+    This folder is used by some large dataset loaders to avoid downloading the
+    data several times.
+
+    By default the data directory is set to a folder named 'skrub_data' in the
+    user home folder.
+
+    Alternatively, it can be set programmatically by giving an explicit folder
+    path. The '~' symbol is expanded to the user home folder.
+
+    If the folder does not already exist, it is automatically created.
+
+    Parameters
+    ----------
+    data_home : str, default=None
+        The path to skrub data directory. If `None`, the default path
+        is `~/skrub_data`.
+
+    Returns
+    -------
+    data_home: str or path-like, default=None
+        The path to skrub data directory.
+    """
+    if data_home is None:
+        data_home = Path("~") / "skrub_data"
```

Comment:
It works on Windows: https://docs.python.org/3/library/os.path.html#os.path.expanduser

> On Windows, USERPROFILE will be used if set, otherwise a combination of HOMEPATH and HOMEDRIVE will be used. An initial ~user is handled by checking that the last directory component of the current user’s home directory matches USERNAME, and replacing it if so.